### PR TITLE
Weekly review 2020-11-26

### PR DESCRIPTION
**Pull request by lailabougria**

### CODE REVIEW

File: nservicebus/pipeline/aborting.md [hunk 1]
```diff
@@ -2,25 +2,25 @@
...
 component: Core
-reviewed: 2019-01-15
+reviewed: 2020-11-26
 ---
 
 
 ## From inside a Handler
 
-From the context of a Handler further Handler execution can be aborted by calling the `DoNotContinueDispatchingCurrentMessageToHandlers()` method. This method instruct the bus not to pass the current message on to subsequent handlers in the pipeline. This is often used by authentication and authorization handlers when those checks fail.
+From the context of a Handler further handler execution can be aborted by calling the `DoNotContinueDispatchingCurrentMessageToHandlers()` method. This method instructs the bus not to pass the current message on to subsequent handlers in the pipeline. This is often used by authentication and authorization handlers when those checks fail.
```
Comment (SeanFeldman) [inline nservicebus/pipeline/aborting.md:11]:
```suggestion
From the context of a Handler, further handler execution can be aborted by calling the `DoNotContinueDispatchingCurrentMessageToHandlers()` method. This method instructs the bus not to pass the current message on to subsequent handlers in the pipeline. This is often used by authentication and authorization handlers when those checks fail.
```

File: nservicebus/pipeline/customizing-error-handling_content_core_[6.0,).partial.md [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
 
-NOTE: Behaviors are to meant to to customize recoverability i.e. how many times and under which conditions a message should be re-tried. To read more about customizing recoverability refer to the [recoverability page](/nservicebus/recoverability/).
+NOTE: Behaviors are meant to customize recoverability i.e. how many times and under which conditions a message should be re-tried. To read more about customizing recoverability refer to the [recoverability documentation](/nservicebus/recoverability/).
 
 
 ### Create a new behavior
 
-Implement a new behavior, which extends the `ITransportReceiveContext` context. This context provides details about the message at the transport level. Calling `next()` in the pipeline will invoke the subsequent pipeline processing steps.
+Implement a new behavior, which extends the `ITransportReceiveContext` interface. This context provides details about the message at the transport level. Calling `next()` in the pipeline will invoke the subsequent pipeline processing steps.
```
Comment (SeanFeldman) [inline nservicebus/pipeline/customizing-error-handling_content_core_[6.0,).partial.md:8]: You've changed it in one place but not another. And the interface is a context contract (interface). Perhaps better to keep it as 'context'?
```suggestion
Implement a new behavior, which extends the `ITransportReceiveContext` context interface. This context provides details about the message at the transport level. Calling `next()` in the pipeline will invoke the subsequent pipeline processing steps.
```

File: nservicebus/pipeline/customizing-error-handling_content_core_[6.0,).partial.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
-Custom exception handling can be implemented using pipeline behaviors. To learn more about pipeline and behaviors refer to [manipulate the pipeline with behaviors](/nservicebus/pipeline/manipulate-with-behaviors.md).
+Custom exception handling can be implemented using pipeline behaviors. To learn more about pipeline and behaviors refer to [the documentation on how to manipulate the pipeline with behaviors](/nservicebus/pipeline/manipulate-with-behaviors.md).
```
Comment (SeanFeldman) [inline nservicebus/pipeline/customizing-error-handling_content_core_[6.0,).partial.md:1]: Shorter link.

```suggestion
Custom exception handling can be implemented using pipeline behaviors. To learn more about pipeline and behaviors refer to the documentation on [how to manipulate the pipeline with behaviors](/nservicebus/pipeline/manipulate-with-behaviors.md).
```

**Review by SeanFeldman — COMMENTED** [review]
A few suggestions and good to go, @lailabougria

**Comment by lailabougria** [comment]
Thanks @SeanFeldman !

**Pull request merged**